packages/react/src/provider/use-open-feature-client-status.ts

```diff
@@ -1,15 +1,20 @@
 import { useEffect, useState } from 'react';
 import { useOpenFeatureClient } from './use-open-feature-client';
-import type { ProviderStatus } from '@openfeature/web-sdk';
+import { ProviderStatus } from '@openfeature/web-sdk';
 import { ProviderEvents } from '@openfeature/web-sdk';
+import type { ReactFlagEvaluationOptions } from '../common';
+import { DEFAULT_OPTIONS, useProviderOptions, normalizeOptions, suspendUntilReady } from '../common';
+
+type Options = Pick<ReactFlagEvaluationOptions, 'suspendUntilReady'>
 
 /**
  * Get the {@link ProviderStatus} for the OpenFeatureClient.
+ * @param options
  * @returns {ProviderStatus} status of the client for this scope
  */
-export function useOpenFeatureClientStatus(): ProviderStatus {
+export function useOpenFeatureClientStatus(options?: Options) {
```

Comment:
We call this hook internally from `useWhenProviderReady`; we should probably explicitly disable suspense in the call from that hook so we don't suspend when we don't want to (since that hook also has it's own suspense logic).